game/templates/game/lesson_detail.html

```
@@ -0,0 +1,393 @@
+        disabled
+    >
+        ✓ Completed
+    </button>
+
+    {% else %}
+
+    <a
+        href="{% url 'complete_lesson' lesson.title %}"
+        style="text-decoration:none;"
+    >
+        <button class="complete-btn">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)